pages/about.tsx

```diff
@@ -0,0 +1,20 @@
+import { Heading, Text } from "@chakra-ui/react"
+import AppLink from "../src/components/AppLink"
+import Layout from "../src/layouts/Layout"
+
+const About:React.FC = () => {
+  return (
+    <Layout>
+      <Heading>Welcome to the #skaleverse</Heading>
+      <Text>Skale, Paper, Scissors is lovingly created by a group of product builder on the SKALE network.</Text>
+      {/* TODO: write up on the game? */}
```

Comment:
... is lovingly created by a group of product builder**s**

Minor grammer note